scripts/update_version.sh

```diff
@@ -12,10 +12,8 @@ setuppy=setup.py
 sed -i "s/version=.*\,/version='$newversion',/" "$setuppy"
 
 wininstall=windows-installer.cfg
-wininstallgui=windows-gui-installer.cfg
```

Comment:
the cli and gui windows installers were combined into a single installer in 0.5.0, this script was not updated to reflect that by mistake.